pkg/functions/runner.go

```diff
@@ -164,16 +164,18 @@ func waitFor(job *Job, timeout time.Duration) error {
 	for {
 		select {
 		case <-time.After(timeout):
-			return errors.New("timed out waiting for function to be ready")
+			timeoutErrMsg := fmt.Sprintf("timed out waiting for function to be ready for %s", timeout)
+			return errors.New(timeoutErrMsg)
```

Comment:
Can you instead just return `ErrRunTimeout` here?

https://github.com/knative/func/blob/18fe6430a185cd21cc1b511720cae40b9eadbbb5/pkg/functions/errors.go#L17

Reply:
ErrRunTimeout seems has no option to actually include the timeout as the message is hardcoded? How do you knwo what was the value of timeout parameter to waitFor - that may be very useful when debugging?